### DISCUSSION TITLE
Particle links are not shown in build version of astro js website

### DISCUSSION THREAD

**Discussion by sNishant011** [matteobruni/tsparticles#4924] [Q&A]
The lines (links) connecting the circles are shown in dev mode but not shown in build mode.
**In Development mode:**

![image](https://user-images.githubusercontent.com/49269819/228897811-543c948c-55af-4141-ad5e-eb327be80e2e.png)

**In Build mode:**

![image](https://user-images.githubusercontent.com/49269819/228898067-2235fdb3-5e14-407b-98ba-895734126ec0.png)

I am using `astro-particles`.

Here is my config:
```
{
  background: {
    color: {
      value: "#ffffff",
    },
  },
  fpsLimit: 60,
  fullScreen: {
    enable: true,
    zIndex: -1,
  },
  particles: {
    color: {
      value: "#74c043",
    },
    links: {
      color: "#74c043",
      distance: 150,
      enable: true,
      opacity: 0.5,
      width: 1,
    },
    collisions: {
      enable: true,
    },
    move: {
      direction: "none",
      enable: true,
      outMode: "bounce",
      random: false,
      speed: 2,
      straight: false,
    },
    number: {
      density: {
        enable: true,
        value_area: 800,
      },
      value: 80,
    },
    opacity: {
      value: 0.5,
    },
    shape: {
      type: "circle",
    },
    size: {
      random: true,
      value: 5,
    },
  },
}
```

**Comment by matteobruni** [comment]
It seems an Astro component issue, please open an issue in the [astro](https://github.com/tsparticles/astro) repository with a reproducible sample
